snippets/pipeLog.md

```diff
@@ -0,0 +1,13 @@
+### pipeLog
+
+Use `console.log` in a pipeline as this function simply encloses it and returns the passed value. This is especially useful for debugging when you want to log a variable's value before its usage.
+
+Logs a value and returns it.
+
+```js
+const pipeLog = data => console.log(data) || data
```

Comment:
Semistandard dictates adding a semicolon (`;`) at the end of the statement.

Reply:
I am also not entirely certain that this is the only way to achieve this. I think something along the lines of `console.log(data), data` would have the same effect.

Reply:
as stated above, a comma may actually be the preferred way to handle this.

Reply:
I didn't know about the comma operator (every time, it's a good time to learn). Now you mention it, it totally makes sense. But I get an `Unexpected use of comma operator` from es-lint while using

```js
const debug = obj => (console.log(obj), obj);
```

I can't find any help on the [`no-sequence`](https://eslint.org/docs/rules/no-sequences) page of es-lint docs.

Reply:
Did you read through the whole docs page? 😉 ``
Rule Details

This rule forbids the use of the comma operator, with the following exceptions:

    In the initialization or update portions of a for statement.
    If the expression sequence is explicitly wrapped in parentheses.